public/app/percona/settings/components/Advanced/Advanced.test.tsx

```diff
@@ -198,7 +228,24 @@ describe('Advanced::', () => {
 
     // expect(spy.calls.mostRecent().args[0].body.stt_check_intervals).toBeDefined();
     expect(spy).toHaveBeenLastCalledWith(
-      expect.objectContaining({ body: expect.objectContaining({ stt_check_intervals: expect.anything() }) })
+      expect.objectContaining({
+        body: expect.objectContaining({
+          advisor_run_intervals: {
+            frequent_interval: '14400s',
+            rare_interval: '280800s',
+            standard_interval: '86400s',
+          },
+          data_retention: '6048000s',
```

Comment:
Same question as above, but I would leave the check for specific values in `advisor_run_intervals`.